packages/orbit-components/src/ButtonMobileStore/ButtonMobileStore.stories.tsx

```diff
@@ -42,20 +42,27 @@ Default.story = {
 };
 
 export const VisualTest = () => {
+  const languages = Object.values(LANGUAGE);
   return (
-    <div className="space-x-xs flex">
-      <ButtonMobileStore
-        onClick={action("clicked")}
-        href="#"
-        type="appStore"
-        alt="Download on the App Store"
-      />
-      <ButtonMobileStore
-        onClick={action("clicked")}
-        href="#"
-        type="googlePlay"
-        alt="Download on the Google Play"
-      />
+    <div className="space-x-xs">
+      {languages.map(lang => (
+        <React.Fragment key={lang}>
+          <ButtonMobileStore
+            onClick={action("clicked")}
+            lang={lang}
+            href="#"
+            type="appStore"
+            alt="Download on the App Store"
+          />
+          <ButtonMobileStore
+            onClick={action("clicked")}
+            lang={lang}
+            href="#"
+            type="googlePlay"
+            alt="Download on the Google Play"
+          />
+        </React.Fragment>
+      ))}
```

Comment:
Shall we run Chromatic with only this change first? 🤔

Reply:
It might not be needed, as the component is super simple. I'm fine with relying on our eyes on this one and just run the visual tests afterwards.

Reply:
Daniel left this ticket with the comment: "Visual tests need to be executed and are expected to fail because I have added more variations. Accept the changes before merging." ;)